src/server/services/wms/qgswmsrendercontext.cpp

```diff
@@ -648,36 +649,42 @@ bool QgsWmsRenderContext::isValidWidthHeight() const
   return true;
 }
 
-QgsRectangle QgsWmsRenderContext::mapExtent() const
+double QgsWmsRenderContext::mapTileBuffer(int mapWidth) const
```

Comment:
Consider setting up the pre-commit hook to make your life easier ;)

Reply:
Yep, done.